.devops/code-review-pipelines.yml

```diff
@@ -54,22 +48,6 @@ stages:
             npx oval validate -p openapi/index.yaml
           displayName: 'Validate API specification'
 
-      - job: danger
-        condition: 
-          and(
-            succeeded(),
-            ne(variables['DANGER_GITHUB_API_TOKEN'], 'skip')
-          )
-        steps:
-          - template: templates/node-job-setup/template.yaml@pagopaCommons
-
-          - bash: |
-              yarn danger ci
```

Comment:
Remove danger from dependencies

Reply:
We can also remove `Dangerfile.ts` from `.eslintrc`